### PR TITLE
Let the custom model take advantage of the namespace

### PR DESCRIPTION
**Pull request by uyab**
Currently this package does not support namespace, but let the custom model use it as per its own needs.

### CODE REVIEW

**Comment by freekmurze** [comment]
Ah, here is the PR already :-)

File: src/TranslationLoaders/TranslationLoader.php [hunk 1]
```diff
@@ -12,5 +12,5 @@
      *
      * @return array
      */
-    public function loadTranslations(string $locale, string $group): array;
```
Comment (freekmurze) [inline src/TranslationLoaders/TranslationLoader.php]: This is a breaking change. If these were to be merged, all users of this package would receive errors that the function signature has been changed. Make that parameter optional.

Comment (uyab) [reply src/TranslationLoaders/TranslationLoader.php]: Even if I make it optional, it is still a breaking change.

Comment (freekmurze) [reply src/TranslationLoaders/TranslationLoader.php]: indeed...

**Comment by freekmurze** [comment]
Holding off this one for now, I'll take a look again when creating a new major version.

**Comment by dehood** [comment]
@freekmurze is there any plans to fix this?

**Comment by tonyyb** [comment]
Hi @freekmurze,

Are you interested for a PR with namespace support and a command to sync translations from files to database or are you already working on a major version ?

Otherwise I just fork an create other version of package.

(after that i would publish a Nova Tool to edit translations)

Best regards.

**Pull request closed without merging**